tests/e2e_test/utils_test.go

```diff
@@ -202,3 +230,108 @@ func GetKymaStateMetricCount(ctx context.Context, kymaName, state string) (int,
 
 	return 0, nil
 }
+
+func GetManifestObjectKey(ctx context.Context, k8sClient client.Client, kymaName, templateName string) (
```

Comment:
you can get manifest CR and resource CR information under `kyma.spec.status.modules`

an example:
```
manifest:
      apiVersion: [operator.kyma-project.io/v1beta2](http://operator.kyma-project.io/v1beta2)
      kind: Manifest
      metadata:
        generation: 5
        name: 31902256-3586-4a02-8e79-80c24c23c690-btp-operator-4128321918
        namespace: kcp-system
    name: btp-operator
    resource:
      apiVersion: [operator.kyma-project.io/v1alpha1](http://operator.kyma-project.io/v1alpha1)
      kind: BtpOperator
      metadata:
        name: btpoperator
        namespace: kyma-system
```